src/sql/expr.rs

```diff
@@ -12,26 +12,30 @@ use std::mem;
 
 use failure::ensure;
 
-use expr as dataflow_expr;
 use ore::collections::CollectionExt;
 use repr::*;
 
 use crate::Params;
 
 // these happen to be unchanged at the moment, but there might be additions later
-pub use dataflow_expr::{
+pub use expr::{
     AggregateFunc, BinaryFunc, ColumnOrder, NullaryFunc, UnaryFunc, UnaryTableFunc, VariadicFunc,
 };
 
+/// This file houses a representation of a SQL plan that is parallel to that found in
+/// src/expr/relation/mod.rs, but represents an earlier phase of planning. It's structurally very
+/// similar to that file, with some differences which are noted below. It gets turned into that
+/// representation via a call to decorrelate().
```

Comment:
Rust tip: Make this comment the first thing in the file, and make the lines start with `//!` instead of `///`. Then it'll be the module's documentation comment, and appear on api.mtrlz.dev if the module is public. (Not sure if this one is, but it is logically a module doc comment, so might as well be one even if rustdoc won't render it as such.)